README.md

```diff
@@ -1,3 +1,130 @@
 # You need a fahrschein to use the (nakadi event) bus
```

Comment:
> And do not forget the banner.

What's the license on that one? I was thinking of using https://commons.wikimedia.org/wiki/File:Bundesarchiv_Bild_183-1990-0104-025,_BVG-Fahrscheine.jpg which is under a  Creative Commons Attribution-Share Alike license.